libcxx/utils/ci/run-buildbot

```diff
@@ -320,6 +320,7 @@ generic-gcc)
                    -DLIBCXXABI_ENABLE_WERROR=NO \
                    -DLIBUNWIND_ENABLE_WERROR=NO
     check-runtimes
+    check-abi-list
```

Comment:
Can you file an issue to track the addition of `check-abi-list` to the GCC builds? I think this is worth doing and it's definitely not going to be a slam dunk, so we'll want an issue to track it.